ntbs-service/wwwroot/source/Components/ValidateTravelOrVisit.ts

```diff
@@ -107,9 +97,10 @@ const ValidateTravelOrVisit = Vue.extend({
         getTravelOrVisitVariables: function () {
             const variables: TravelOrVisitVariables = {
                 totalNumberOfCountries: this.$refs["totalNumberOfCountries"].value,
-                country1Id: this.$refs["country1Id"].value,
-                country2Id: this.$refs["country2Id"].value,
-                country3Id: this.$refs["country3Id"].value,
+                // Because of autocomplete, select inputs are hidden and we wrap them in a div that we can reference
+                country1Id: this.$refs["country1Id"].getElementsByTagName("select")[0].value,
```

Comment:
I started working on this page first and this worked - it gave me the value on the original select item. However, now it's always blank.. I really don't know what's changed as I merged all my commits :(